tests/rkt_image_list_test.go

```diff
@@ -77,14 +78,17 @@ func TestImageSize(t *testing.T) {
 
 	imageListCmd := fmt.Sprintf("%s image list --no-legend --full", ctx.Cmd())
 
-	// check that the printed size is the same as the actual image size
-	expectedStr := fmt.Sprintf("(?s)%s.*%d.*", imageHash, imageSize)
+	// if we don't support overlay fs, we don't render the image on fetch
+	if !common.SupportsOverlay() {
```

Comment:
I am honored by the dedication :-D

If looking for inspiration, please take a look at https://github.com/coreos/rkt/pull/2007#issuecomment-213798710, https://github.com/s-urbaniak/rkt/commit/5a76a26321245b46cc890a4059ec2a3938574002 ;-)